src/features/hotspots/details/HotspotDetailChart.tsx

```diff
@@ -36,44 +98,48 @@ const HotspotDetailChart = ({
         padding="l"
         borderRadius="m"
         flexDirection="row"
-        justifyContent="space-between"
+        justifyContent={loading ? 'center' : 'space-between'}
         alignItems="center"
+        height={136}
       >
-        {percentage ? (
-          <Box>
-            <Text variant="body3" color="grayLightText" marginBottom="s">
-              {t('hotspot_details.pass_rate')}
-            </Text>
-            <Text variant="light" fontSize={34} color="black" marginBottom="s">
-              {`${focusedData ? focusedData.up : percentage}%`}
-            </Text>
-          </Box>
+        {loading ? (
+          <ActivityIndicator size="small" />
```

Comment:
I believe for activity indicators to show on Android they need a `color` prop. Something like...
`<ActivityIndicator color={colors.grayBox} />`

Reply:
Good point, updated